core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -448,6 +448,51 @@ describe('Contributions and review component', function() {
       expect(ctrl.contributionsDataLoading).toBe(false);
     });
 
+    it('should handle undefined translation opportunities' +
+      ' in review tab', function() {
```

Comment:
Be more specific with the expected behavior? Maybe something like "should not throw error for undefined..." Ditto below.